modules/core/src/main/java/org/apache/ignite/internal/processors/security/SecurityUtils.java

```diff
@@ -120,6 +117,24 @@ public static void restoreDefaultSerializeVersion() {
         return srvcPerms;
     }
 
+    /**
+     * Gets a current security subject id for communication routines.
+     *
+     * @return Current security subject id if security is enabled and subject id doesn't equal local node id otherwise
```

Comment:
Current security subject id if security is enabled and subject id doesn't equal to local node id, otherwise {@code null}.